Artifacts/PgSql/Data/Security/1060-RemoveRedundantBootstrapPermissions.sql

```diff
@@ -0,0 +1,49 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DO language plpgsql $$
+DECLARE
+    educationOrganizationsResourceClaimId INT;
+	claimSet_Id INT;
```

Comment:
Here again... convert all <kbd>TAB</kbd>s to spaces.

Reply:
yes , I formatted with indent . Thanks